fhir-server/liberty-config/configDropins/overrides/datasource-postgresql.xml

```diff
@@ -0,0 +1,18 @@
+<server>
+    <!-- ============================================================== -->
+    <!-- TENANT: default; DSID: default; TYPE: read-write               -->
+    <!-- ============================================================== -->
+    <dataSource id="fhirDatasourceDefaultDefault" jndiName="jdbc/fhir_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true">
+        <jdbcDriver javax.sql.XADataSource="org.postgresql.xa.PGXADataSource" libraryRef="fhirSharedLib"/>
+            <properties.postgresql
+                 serverName="localhost"
+                 portNumber="5432"
+                 databaseName="fhirdb"
+                 user="fhirserver"
+                 password="change-password"
+                 currentSchema="fhirdata"
+             />
+        />
```

Comment:
How does the multiple db work? Is there a doc update?

Reply:
Multiple databases can be defined (see the configDropins/overrides in the fhir-server liberty-config dir. Now that this is passing SIT, I'm going to work on updating the documentation.

Reply:
As discussed with the team this morning, the legacy behavior is default. You have to actively switch off the proxy datasource in fhir-server-config in order to get the new behavior.

Reply:
Draft update to FHIRServerUsersGuide.md just added to this PR.